pkgs/top-level/python-packages.nix

```diff
@@ -6090,6 +6090,10 @@ in {
   fastapi = callPackage ../development/python-modules/fastapi { };
 
   stringcase = callPackage ../development/python-modules/stringcase { };
+
+  runway-python = callPackage ../development/python-modules/runway-python { };
+
+  wget = callPackage ../development/python-modules/wget { };
```

Comment:
Please create a separate commit for each package.

Reply:
updated